File: project/app/src/main/java/org/owntracks/android/ui/welcome/intro/IntroFragment.java
package org.owntracks.android.ui.welcome.intro;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import org.owntracks.android.R;
import org.owntracks.android.databinding.UiWelcomeIntroBinding;
import org.owntracks.android.ui.base.BaseSupportFragment;
import org.owntracks.android.ui.base.viewmodel.NoOpViewModel;
import org.owntracks.android.ui.welcome.WelcomeFragmentMvvm;

public class IntroFragment extends BaseSupportFragment<UiWelcomeIntroBinding, NoOpViewModel> implements WelcomeFragmentMvvm.View {
    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return setAndBindContentView(inflater, container, R.layout.ui_welcome_intro, savedInstanceState);
    }

    @Override
    public void onNextClicked() {
    }

    @Override
    public boolean isNextEnabled() {
        return true;
    }

    @Override
    public void onShowFragment() {

    }
}

File: project/app/src/main/java/org/owntracks/android/ui/welcome/intro/IntroFragment.kt
package org.owntracks.android.ui.welcome.intro

import dagger.hilt.android.AndroidEntryPoint
import org.owntracks.android.R
import org.owntracks.android.databinding.UiWelcomeIntroBinding
import org.owntracks.android.ui.welcome.BaseWelcomeFragment
import javax.inject.Inject

@AndroidEntryPoint
class IntroFragment @Inject constructor() :
        BaseWelcomeFragment<UiWelcomeIntroBinding>(R.layout.ui_welcome_intro) {
    override fun onResume() {
        super.onResume()
        activityViewModel.nextEnabled.postValue(true)
        activityViewModel.doneEnabled.postValue(false)
    }
}